DIAM-1949: Errors with tickets' images interactions

kolesnik-sashko opened the pull request:
- Edited csrf token name
- Changed upload images directory for attachments
- FIxed error with generated preview link with non existent route
- Added upload directories to gitignore

Commit:
DIAM-1949: Errors with tickets' images interactions
- Edited csrf token name
- Changed upload images directory for attachments
- FIxed error with generated preview link with non existent route
- Added upload directories to gitignore

krasilich commented on src/Diamante/DeskBundle/Infrastructure/Attachment/FileStorage/LocalFileStorageService.php:
```diff
@@ -69,15 +70,27 @@ public function remove($filename)
+            . Attachment::PATH_TO_ATTACH_DIR;
+        if (!$fs->exists($attachmentUploadDirPath)) {
+            $fs->mkdir($attachmentUploadDirPath);
+        }
```
Attachments directory should be created before installation, there is also a check for this
`https://github.com/eltrino/diamantedesk-application/blob/oro-3.1-upgrade/var/DiamanteDeskRequirements.php:20`
Also, note that we have attachment directory which is used by Oro and attachments for Diamante purposes.

kolesnik-sashko replied:
As I can see in this file we check '/var/attachments' directory, not '/src/attachments'. And what about if we have already installed application?

Commit:
Merge branch 'oro-3.1-upgrade' of github.com:eltrino/diamantedesk-application into DIAM-1949

Commit:
DIAM-1949: Errors with tickets' images interactions
- Added new data_root dir to liip_imagine configuration
- Changed upload path to var/attachments

Commit:
DIAM-1949: Errors with tickets' images interactions
- Removed unusable class
